src/App.tsx

```diff
@@ -1,61 +1,129 @@
+import { SetStateAction, useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './components/TodoInfo/TodoInfo';
 
 export const App = () => {
+  const todosWithUsers = todosFromServer.map(todo => {
+    const user = usersFromServer.find(u => u.id === todo.userId) || null;
+
+    return {
+      ...todo,
+      user,
+    };
+  });
+
+  const [todos, setTodos] = useState<Todo[]>(todosWithUsers);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('');
+  const [titleError, setTitleError] = useState('');
+  const [userError, setUserError] = useState('');
+
+  const handleTitleChange = (event:
+  { target: { value: SetStateAction<string>; }; }) => {
+    setTitle(event.target.value);
+    if (titleError) {
+      setTitleError('');
+    }
+  };
+
+  const handleUserChange = (event:
+  { target: { value: SetStateAction<string>; }; }) => {
```

Comment:
Those types for events look really messy here, try do do it in a similar way as I suggested in `handleSubmit`